packages/integration-tests/src/test-projects/advanced-auth-and-functions/amplify/function.ts

```diff
@@ -24,6 +26,14 @@ export const funcNoMinify = defineFunction({
   },
 });
 
+export const funcProvided = defineFunction((scope) => {
+  return new NodejsFunction(scope, 'funcProvided', {
+    entry:
+      './packages/integration-tests/src/test-projects/advanced-auth-and-functions/amplify/func-src/handler_provided.ts',
```

Comment:
Actually I tested these on unit tests, this path should be like this, but I don't know about in integration test, if it's run without an error, that's cool

Reply:
Ok. I can see what's missing.

We need to add `funcProvided` here https://github.com/aws-amplify/amplify-backend/blob/b574bce26ec2d55a4391aeceecc1d4957e79fba3/packages/integration-tests/src/test-projects/advanced-auth-and-functions/amplify/test_factories.ts#L10 . so that it actually becomes member of test project.